docs/api-ref.md

```diff
@@ -1374,6 +1374,55 @@ Returns the `JobItem` requested.
 ```
 
 
+<br>
+<br>
+
+---
+
+## Metadata
+
+Using the `metadata` endpoint, one can use GraphQL to query metadata on, e.g., workbooks, datasources, the fields contained within them, and the relationships between them.
+
+### Metadata methods
+
+
+The Metadata methods are based upon the [Metadata API](https://help.tableau.com/current/api/metadata_api/en-us/index.html) REST methods.
```

Comment:
They aren't really REST methods.

I would say something like "The Metadata methods call the Metadata API, which is a GraphQL interface" or something similar